server/src/test/java/com/objectcomputing/checkins/services/pulseresponse/PulseResponseBadArgExceptionTest.java

```diff
@@ -9,7 +10,7 @@
     @Test
     void testExceptionMessage() {
         final String message = "Hello world";
-        PulseResponseBadArgException argException = new PulseResponseBadArgException(message);
+        BadArgException argException = new BadArgException(message);
```

Comment:
Should we delete this file instead?

Reply:
Agreed. BadArgException is exercised in other ways.

Reply:
deleted the file.